.circleci/config.yml

```diff
@@ -58,11 +58,19 @@ jobs:
           name: "Link packages"
           command: |
             yarn install
+      - run:
+          name: Workaround for https://github.com/GoogleChrome/puppeteer/issues/290
+          command: sh ./scripts/workaround-puppeteer-issue-290.sh
       - run:
           name: "Build react kitchen-sink"
           command: |
             cd examples/cra-kitchen-sink
             yarn build-storybook
+      - run:
+          name: "Run image snapshots on react kitchen-sink"
+          command: |
+            cd examples/cra-kitchen-sink
+            yarn test-image-snapshots
```

Comment:
Can we use built versions for integration tests as well?

Reply:
Yep I was thinking about doing so.